examples/pass-run-time-values-to-tools.ipynb

```diff
@@ -5,18 +5,13 @@
    "id": "51466c8d-8ce4-4b3d-be4e-18fdbeda5f53",
    "metadata": {},
    "source": [
-    "# How to pass runtime values to tools\n",
+    "# How to pass graph state to tools\n",
```

Comment:
could you please the notebook name too? and put it into tool calling section https://langchain-ai.github.io/langgraph/how-tos/tool-calling/

Reply:
changing notebook name will break existing links